scripts/hail_batch/variant_selection/hgdp_1kg_tob_wgs_variant_selection.py

```diff
@@ -65,8 +65,6 @@ def query(output):  # pylint: disable=too-many-locals
         hgdp1kg_tobwgs_joined.GT, r2=0.1, bp_window_size=500000
     )
     pruned_variant_table = pruned_variant_table.cache()
```

Comment:
This line still persists the table in memory, did you mean to remove it?

```suggestion
```

Reply:
thanks for that catch. Yes, I forgot to remove that one